reporter/html.js

```diff
@@ -707,8 +710,18 @@ QUnit.testDone(function( details ) {
 		}
 	}
 
+	if (config.collapse !== collapseCheck) {
+		collapse = config.collapse;
+	}
+
 	if ( bad === 0 ) {
 		addClass( assertList, "qunit-collapsed" );
+	} else {
+		if (( collapse === true ) && (collapseFirst === 0 )) {
```

Comment:
also, there are missing spaces between the parenthesis. basically: replace `((` and `))` to `( (` and `) )`